.pre-commit-config.yaml

```diff
@@ -5,12 +5,6 @@ repos:
       - id: black
         files: "^datamodel_code_generator|^tests"
         exclude: "^tests/data"
-  - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
```

Comment:
@KotlinIsland 
Can we keep isort in pre-commit?

Is it difficult?

Reply:
Ruff does isort 🚀